website/docs/integrations/home_assistant.md

```diff
@@ -66,10 +72,10 @@ Don't forget to replace `<teslamate url>` and `<your tesla model>` with correct
         payload_not_available: "false"
     device: &teslamate_device_info
       identifiers: [teslamate_car_1]
-      configuration_url: https://teslamate.zxxz.io/ # update this with your teslamate URL
+      configuration_url: <teslamate url> # update this with your teslamate URL, e.g. https://teslamate.example.com/
       manufacturer: Tesla
-      model: Model 3 # update this with your car model
-      name: Tesla Model 3 # update this with your car name
+      model: Model 3 # <your tesla model>, update this with your car model
+      name: Tesla Model 3 # <your tesla name> update this with your car name
```

Comment:
<your tesla ...> should be in the yml, Model 3 and Tesla Model 3 in the comment as example ?

Reply:
Indeed, same as above, laziness.